analyzers/its/config/SonarLintExclusions/SonarLint.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<AnalysisInput>
+    <Rules>
+    </Rules>
+    <Settings>
+        <Setting>
+            <Key>sonar.cs.ignoreHeaderComments</Key>
+            <Value>true</Value>
+        </Setting>
+        <Setting>
+            <Key>sonar.vbnet.ignoreHeaderComments</Key>
+            <Value>true</Value>
+        </Setting>
+        <Setting>
+            <Key>sonar.inclusions</Key>
+            <Value>**/Included/*.cs</Value>
+        </Setting>
+        <Setting>
+            <Key>sonar.exclusions</Key>
+            <Value>**/ExcludedByExclusion.cs</Value>
```

Comment:
Open question: is it worth adding a test for multiple exclusions/inclusions? For example, adding a new `ExcludedBySecondExclusion.cs` file and having the comma-separated list here
```suggestion
            <Value>**/ExcludedByExclusion.cs,**/ExcludedBySecondExclusion.cs</Value>
```